app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -33,35 +33,36 @@ def supports_authentication?(authtype)
     supported_auth_types.include?(authtype.to_s)
   end
 
-  def self.raw_connect(server, port, username, password, service = "Service")
+  def self.raw_connect_metrics(connect_options)
```

Comment:
I'd rather have the explicit parameters back.  So instead of `connect_options`, use the `server`, `port`, `username`, and `password` all spelled out.

Especially since this is a class level method, I want to be able to easily see what to pass in to build the connection.  As it stands, looking at the code, I can't tell what values to throw into `connect_options` without diving deeper into the OvirtMetrics gem to see what connection options it takes.